apps/cli/scripts/install-debial.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+{
+    set -e
+    SUDO=''
+    if [ "$(id -u)" != "0" ]; then
+      SUDO='sudo'
+      echo "This script requires superuser access to install apt packages."
+      echo "You will be prompted for your password by sudo."
+      # clear any previous sudo permission
+      sudo -k
+    fi
+
+    # run inside sudo
+    $SUDO sh <<SCRIPT
+  set -ex
+
+  # if apt-transport-https is not installed, clear out old sources, update, then install apt-transport-https
+  dpkg -s apt-transport-https 1>/dev/null 2>/dev/null || \
+    (echo "" > /etc/apt/sources.list.d/sunodo.list \
+      && apt-get update \
+      && apt-get install -y apt-transport-https gnupg2 curl)
+
+  # add sunodo repository to apt
+  echo "deb https://cli.sunodo.io/channels/stable/apt ./" > /etc/apt/sources.list.d/sunodo.list
+
+  # install sunodo's release key for package verification
+  curl https://cli.sunodo.io/release.key | apt-key add -
```

Comment:
this method of adding the key is deprecated.
Check this blog post
https://itsfoss.com/apt-key-deprecated/